specification/search/data-plane/Microsoft.Azure.Search.Data/preview/2019-05-06-preview/searchindex.json

```diff
@@ -1103,6 +1199,19 @@
           "$ref": "#/definitions/QueryType",
           "description": "A value that specifies the syntax of the search query. The default is 'simple'. Use 'full' if your query uses the Lucene query syntax."
         },
+        "featuresMode": {
+          "$ref": "#/definitions/FeaturesMode",
+          "description": "A value that specifies whether the results should include scoring features, such as per field similarity. The default is 'disabled'. Use 'enabled' to expose additional scoring features."
+        },
+        "scoringStatistics": {
+          "$ref": "#/definitions/ScoringStatistics",
+          "description": "A value that specifies whether we want to calculate scoring statistics (such as document frequency) globally for more consistent scoring, or locally, for lower latency. The default is 'local'. Use 'global' to aggregate scoring statistics globally before scoring."
+        },
+        "sessionId": {
+          "type": "string",
+          "description": "A value to be used to create a sticky session, which can help getting more consistent results. As long as the same sessionId is used, a best-effort attempt will be made to target the same replica set.",
+          "x-ms-client-name": "SessionId"
```

Comment:
No need for `x-ms-client-name` here. AutoRest is smart about adjusting the case of property names to camelCase, PascalCase, or snake_case depending on the target language.